src/main/java/com/fouristhenumber/utilitiesinexcess/common/blocks/BlockDrum.java

```diff
@@ -0,0 +1,227 @@
+package com.fouristhenumber.utilitiesinexcess.common.blocks;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockContainer;
+import net.minecraft.block.material.Material;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.item.EntityItem;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.StatCollector;
+import net.minecraft.world.World;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.IFluidContainerItem;
+
+import com.fouristhenumber.utilitiesinexcess.common.tileentities.TileEntityDrum;
+
+public class BlockDrum extends BlockContainer {
+
+    int capacity = 16000;
+
+    public BlockDrum() {
+        super(Material.iron);
+        setBlockName("drum");
+        this.setHardness(3.0F);
+        this.setResistance(5.0F);
+        setBlockTextureName("utilitiesinexcess:drum");
+        this.setHarvestLevel("pickaxe", 1);
+    }
+
+    @Override
+    public TileEntity createNewTileEntity(World world, int meta) {
+        return new TileEntityDrum(capacity);
+    }
+
+    @Override
+    public boolean hasTileEntity(int metadata) {
+        return true;
+    }
+
+    @Override
+    public boolean onBlockActivated(World world, int x, int y, int z, EntityPlayer player, int side, float hitX,
+        float hitY, float hitZ) {
+        if (world.isRemote) {
+            return true;
+        }
+        TileEntity tile = world.getTileEntity(x, y, z);
+        if (tile instanceof TileEntityDrum drum) {
+            if (drum.tank != null && drum.tank.getFluid() != null) {
+                player.addChatMessage(
+                    new ChatComponentText(
+                        "In drum: " + drum.tank.getFluid()
```

Comment:
Use translatable text components with language keys and gtnhlib number formatting here

Reply:
Done

Reply:
There is a ChatComponent for translatable strings, the way you did it will only translate to the server's language, not the client's